NorthstarDLL/color.h

```diff
@@ -130,6 +154,44 @@ class Color
 		return newColor;
 	}
 
+	std::string ToANSIColor()
+	{
+		std::string out = "\033[38;2;";
+		out += std::to_string(_color[0]) + ";";
+		out += std::to_string(_color[1]) + ";";
+		out += std::to_string(_color[2]) + ";";
+		out += "49m";
```

Comment:
49 resets the background colour of the text, there shouldn't be any problems with this for now, however if we want to add support for custom background colours (not doable in the source console afaik?) this could cause issues. 

I wouldn't say that this is a problem atm, just something to note, maybe with a comment or something?